packages/flutter/lib/src/rendering/list_wheel_viewport.dart

```diff
@@ -562,34 +626,109 @@ class RenderListWheelViewport
       perspective: _perspective,
     );
 
-    context.pushTransform(
-      // Text with TransformLayers and no cullRects currently have an issue rendering
-      // https://github.com/flutter/flutter/issues/14224.
-      false,
-      offset,
-      _centerOriginTransform(transform),
-      // Pre-transform painting function.
-      (PaintingContext context, Offset offset) {
-        context.paintChild(
-          child,
-          offset + new Offset(
-            untransformedPaintingCoordinates.dx,
-            // Paint everything in the center (e.g. angle = 0), then transform.
-            -_topScrollMarginExtent,
-          ),
-        );
-      },
-    );
+    void _paintCylindrically(PaintingContext context, Offset offset) {
```

Comment:
Might be clearer if this was a separate function instead of nesting it.

Reply:
Leaving the function outside means I have to pass more arguments to it. Please review after I update the code.